server/errors.go

```diff
@@ -0,0 +1,28 @@
+package server
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+	"github.com/prior-it/apollo/core"
+)
+
+func DefaultErrorHandler(apollo *Apollo, err error) {
+	apollo.Error("Server error", "error", err)
```

Comment:
Die "[ERROR]" aan het begin van de error message (zoals in de vorige error handler) vond ik wel duidelijk. Enige reden dat je die hebt weggedaan?